easybuild/easyconfigs/p/PyTorch/PyTorch-1.11.0_skip_failing_ops_tests.patch

```diff
@@ -0,0 +1,26 @@
+diff -Nru pytorch-1.11.0-rc3.orig/torch/testing/_internal/common_methods_invocations.py pytorch-1.11.0-rc3/torch/testing/_internal/common_methods_invocations.py
```

Comment:
@casparvl comment to clarify what this patch is doing and where it comes from is missing

The easyconfigs test suite is supposed to catch this, but apparently it doesn't in this case, I'll have to check where that is... 🤔

Reply:
Description added